coding-concepts.js

```diff
@@ -40,7 +60,13 @@ var onlyOdds = [11, 12, 13, 14, 15].filter(value => value % 2 !== 0)
 // console.log(onlyOdds)
 
 // a) Your answer:
+// I'm expecting onlyOdds to be modified here but I THINK the date needs to be stored in another variable if it is to be retrieved. I'll say that the output will be the original array and not the filtered items due to not being assigned a value in a new variable.
```

Comment:
By the way, how you answer the questions as a conversation is superb! It will help any job interviewer connect with you faster. Keep this approach and write exactly how you think the console.log appears as discussed earlier.